Sources/BraintreeDropIn/Custom Views/BTPaymentSelectionHeaderView.h

```diff
@@ -0,0 +1,21 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol BTPaymentSelectionHeaderViewDelegate;
+
+@interface BTPaymentSelectionHeaderView : UITableViewHeaderFooterView
```

Comment:
This is used to display the "RECENT" and "OTHER" labels as well as the "Edit" button.

Reply:
I think a comment like this might be useful in the file itself; at first read of the file name I was a bit confused what this was going t be used for

Reply:
✔️